retriever/lib/get_opts.py

```diff
@@ -59,9 +64,9 @@
 install_parser.add_argument('--debug', help='run in debug mode', action='store_true')
 install_parser.add_argument('--not-cached', help='overwrites local cache of raw data', action='store_true')
 download_parser.add_argument('dataset', help='dataset name').completer = ChoicesCompleter(script_list)
-ls_parser.add_argument('-l', help='verbose list of datasets containing following keywords '
+ls_parser.add_argument('-l', help='search datasets with either specific license(s) or keyword(s)'
                                   '(lists all when no keywords are specified)',
```

Comment:
This line should be deleted since it no longer holds true. `retriever ls -l` used to return the entire list of datasets with more information on all of them, but I'm not sure when that was taken out, or if we want to put that capability back in.

Reply:
This comment is specifically only in regards to line 68: 

`(lists all when no keywords are specified)`